src/components/app/details/triggerView/cdMaterial.tsx

```diff
@@ -596,7 +610,7 @@ export class CDMaterial extends Component<CDMaterialProps, CDMaterialState> {
     }
 
     renderConfigDiffStatus() {
-        const _canReviewConfig = this.canReviewConfig()
+        const _canReviewConfig = this.canReviewConfig() && this.state.recentDeploymentConfig !== null
```

Comment:
check for this.state.recentDeploymentConfig is already present in canReviewConfig method. do we still need check it again?

Reply:
Actually the _canReviewConfig variable is used locally to only render the state which shows the checking difference, (green or red colour depending on difference in config) but the above function canReviewConfig() was used at multiple places which needed the this.state.recentDeploymentConfig to be null which is even if there are no previous deployments I want to review the config, but in case of _canReview I needed an opposite boolean to not show the diff state(green or red state) in case there are no previous deployments.